src/docs/product/alerts-notifications/issue-alerts.mdx

```diff
@@ -32,26 +51,22 @@ Actions are the configurations that determine what will happen when an alert is
 
   If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
 
-To change the frequency of issue alert emails, update your alert settings [digests](/product/alerts-notifications/alert-settings/#digests).
+#### Issue Owners
 
-### Issue Owners
-
-By default, [issue owners](/product/error-monitoring/issue-owners/) receive alert emails. If an issue owner is not configured or not found, the email will either not send or send to all project members as defined in **Project Settings > Issue Owners**.
+[Issue owners](/product/error-monitoring/issue-owners/) can receive alert emails. If an issue owner is not configured or not found, the email will either not send or send to all project members as defined in **Project Settings > Issue Owners**.
 
 ![A toggle indicating if all users are issue owners or not.](issue_owners2.png)
 
 ## Rate Limit
 
-The rate limit determines how frequently an issue alerts. After an issue triggers an alert, Sentry won't check the conditions and won't execute the actions for that issue until the rate limit period passes. The limit is set to perform the action according to one of these intervals:
+The rate limit controls how often the alert rule can be triggered for a particular issue. If alert conditions match an issue, actions are only executed if no other actions have been executed within the rate limit period. The limit is set to perform the action according to one of these intervals:
```

Comment:
"only executed if no other actions have been executed" is confusing .... suggest "if they have not been executed". actions are not differentiated, either all or none of them execute